cli/lib/kontena/cli/nodes/labels/remove_command.rb

```diff
@@ -12,7 +12,7 @@ class RemoveCommand < Kontena::Command
     def execute
       node = client.get("nodes/#{current_grid}/#{node_id}")
       data = { labels: Array(node['labels']).reject {|label| label_list.include?(label) } }
-      client.put("nodes/#{current_grid}/#{node['id']}", data)
+      client.put("nodes/#{current_grid}/#{node['name']}", data)
```

Comment:
This is `kontena node label remove`, not `kontena node remove` :)

> Shouldn't `node['id']` work here?

Sure, without the `current_grid`. The `#{current_grid}/#{name}` just matches what all the other commands use. Could potentially change all of the commands to use `nodes/#{node['id']}`?